### DISCUSSION TITLE
Canadian postal codes returning NA for lat/long

### DISCUSSION THREAD

**Discussion by macsmith26** [jessecambon/tidygeo#144] [Q&A]
Hey, just wanted to know if Canadian postal codes work with `geo()`? I'm getting `NA`'s when only supplying Canadian postal codes to the `postalcode` argument, but US zip codes seem to work fine.

**Comment by jessecambon** [comment]
Hi @macsmith26, they should. Which service are you using (ie. `geo(method = ??)` ). Do you have an example query?
